enhancements/sig-architecture/64-placementStrategy/README.md

```diff
@@ -0,0 +1,220 @@
+# Placement Strategy
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+The proposed work would define a new field in the Placement API (placementStrategy) that will let the end user able to break down the selected managed clusters to groups and define canary clusters.
```

Comment:
The placement APIs is to provide a definition on how to select a set of ManagedClusters in multiple ManagedClusterSets. The end user can use it in multiple scenarios including upgrade. canary clusters sounds like only used in upgrade scenarios, right?  not sure if it is proper to define a specified property in a common API spec.

Reply:
okay canary definition has been removed and add affinity cluster selection as more generic way to define a special subset.